config/initializers/constants.rb

```diff
@@ -1,4 +1,8 @@
 # frozen_string_literal: true
 
-SEARCH_ENGINES = %w[BingV6 BingV7 Google SearchGov].freeze
+ADMIN_EMAIL_ADDRESS = Rails.application.secrets.organization[:admin_email_address].freeze
 DEFAULT_USER_AGENT = Rails.application.secrets.organization[:default_user_agent].freeze
+DELIVER_FROM_EMAIL_ADDRESS = 'no-reply@support.digitalgov.gov'.freeze
```

Comment:
You can remove the `freeze` from the strings, as the `frozen_string_literal: true` setting will freeze them. (Rubocop would normally flag this, except that we've disabled Rubocop for the `config/` directory: https://docs.rubocop.org/rubocop/cops_style.html#styleredundantfreeze)

Reply:
Good reminder that rubocop doesn't run against everything.  Also added in missing newline.